kubectl-rook-ceph.sh

```diff
@@ -30,10 +30,16 @@ function print_usage () {
   echo "  -h, --help                  : output help text"
   echo "  -n, --namespace='rook-ceph' : the namespace of the CephCluster"
   echo "COMMANDS"
-  echo "  help           : output help text"
-  echo "  ceph <args>    : call a 'ceph' CLI command with arbitrary args"
+  echo "  help                  : output help text"
+  echo "  ceph <args>           : call a 'ceph' CLI command with arbitrary args"
+  echo "  rbd <args>            : call a 'rbd' CLI command with arbitrary args"
   echo "  operator <subcommand>..."
-  echo "    restart      : restart the Rook-Ceph operator"
+  echo "    restart             : restart the Rook-Ceph operator"
+  echo "    set <property> <value>   : Set the property in the rook-ceph-operator-config configmap."
+  echo "  rook <subcommand>..."
+  echo "    version             : print the version of Rook"
+  echo "    status <args>       : print the phase of CR in the namespace. Default is CephCluster CR status"
+  echo "  get-mon-endpoints     : print mon endpoints"
```

Comment:
How about this pattern to get info? It seems more consistent with other commands
```suggestion
  echo "  get <subcommand>"
  echo "    mons     : print mon endpoints"
```